image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       labels:
         app: central
         {{- include "srox.podLabels" (list . "deployment" "central") | nindent 8 }}
+        openshift.io/required-scc: {{ include "srox.globalResourceName" (list . "stackrox-central") }}
```

Comment:
This still needs to be checked what is the optima value. You can refer to the table in this WIP-PR (when it is updated): https://github.com/stackrox/stackrox/pull/7671